seqr/views/apis/individual_api.py

```diff
@@ -399,9 +396,7 @@ def add_or_update_individuals_and_families(project, individual_records):
             set_patient_hpo_terms(project, individual.phenotips_eid, record.get('hpoTerms'), is_external_id=True)
 
         if not individual.display_name:
-            individual.display_name = individual.individual_id
-
-        individual.save()
+            update_seqr_model(individual, display_name=individual.individual_id)
 
         _deprecated_update_original_individual_data(project, individual)
```

Comment:
if we are using `update_seqr_model` we shouldn't also need `_deprecated_update_original_individual_data`

Reply:
multiple models are updated here so will as is